template/README.md

```diff
@@ -19,45 +19,101 @@ Search for and replace the following placeholders within this file:
-1. Setup a SSH key that can be used for GitHub and the Acquia Cloud (you CAN use the same key).
+2. Ensure that your computer meets the minimum installation requirements (and then install the required applications). See the [System Requirements](http://blt.readthedocs.io/en/latest/INSTALL/#system-requirements).
+3. Request access to organization that owns the project repo in GitHub (if needed).
+4. Fork the project repository in GitHub.
+5. Request access to the Acquia Cloud Environment for your project (if needed).
+6. Setup a SSH key that can be used for GitHub and the Acquia Cloud (you CAN use the same key).
     1. [Setup GitHub SSH Keys](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/)
-    1. [Setup Acquia Cloud SSH Keys](https://docs.acquia.com/acquia-cloud/ssh/generate)
-1. Clone your forked repository. By default, Git names this "origin" on your local.
+    2. [Setup Acquia Cloud SSH Keys](https://docs.acquia.com/acquia-cloud/ssh/generate)
+7. Clone your forked repository. By default, Git names this "origin" on your local.
     ```
     $ git clone git@github.com:<account>/#GITHUB_PROJECT.git
-1. To ensure that upstream changes to the parent repository may be tracked, add the upstream locally as well.
+    ```
+8. To ensure that upstream changes to the parent repository may be tracked, add the upstream locally as well.
     ```
     $ git remote add upstream git@github.com:#GITHUB_ORG/#GITHUB_PROJECT.git
     ```
-1. Install Composer Dependencies. (Warning: this can take some time based on internet speeds.)
+
+9. Update your the configuration located in the `/blt/blt.yml` file to match your site's needs. See [configuration files](#important-configuration-files) for other important configuration files.
+
+
+----
+# Setup Local Environment.
+
+BLT provides an automation layer for testing, building, and launching Drupal 8 applications. For ease when updating codebase it is recommended to use  Drupal VM. If you prefer, you can use another tool such as Docker, Docksal, Lando, (other) Vagrant, or your own custom LAMP stack, however support is very limited for these solutions.
+1. Install Composer dependencies.
+After you have forked, cloned the project and setup your blt.yml file install Composer Dependencies. (Warning: this can take some time based on internet speeds.)
     ```
     $ composer install
     ```
-1. Setup local environment.
+2. Setup VM with BLT.
+Use BLT to setup the Drupal VM with the configuration from this repositories [configuration files](#important-configuration-files) (one time only).
 
-    BLT requires "some sort" of local environment that implements a LAMP stack. While we provide out of the box templates for Drupal VM, if you prefer you can use another tool such as Docker, Docksal, Lando, (other) Vagrant, or your own custom LAMP stack. BLT works with any local environment, however support is limited for these solutions.
+    ```
+    $ blt vm
```

Comment:
we need to be careful here, because you should only run blt vm if you are using drupal vm. i don't mind some of the rewriting you have done here, but we need to be clear that this is ONLY going to setup drupal vm and should only be used if you truly intend to use that.